### PR TITLE
Mpv: add motion estimation via vapoursynth

### PR DESCRIPTION
**Pull request by lemmi**
this will need a little bit extra work, as some dependencies don't compile for anything but x86{,_64}.

### CODE REVIEW

File: srcpkgs/vapoursynth/template [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Template file for 'vapoursynth'
+pkgname=vapoursynth
+version=R31
+revision=1
+build_style=gnu-configure
+hostmakedepends="base-devel yasm python3.4-devel python-Cython python3.4-Cython"
```
Comment (Vaelatern) [inline srcpkgs/vapoursynth/template]: `-devel` things ought to go in `makedepends` and on no accounts should you depend on `base-devel`

File: srcpkgs/vapoursynth-mvtools/template [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# Template file for 'vapoutsynth-mvtools'
+pkgname=vapoursynth-mvtools
+version=12
+revision=1
+build_style=gnu-configure
+hostmakedepends="base-devel yasm"
+makedepends="fftw-devel vapoursynth zimg-devel"
+depends=""
```
Comment (pullmoll) [inline srcpkgs/vapoursynth-mvtools/template]: Remove this line.

File: srcpkgs/zimg/template [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Template file for 'zimg'
+pkgname=zimg
+version=2.0.4
+revision=1
+wrksrc=zimg-release-${version}
+build_style=gnu-configure
+hostmakedepends="base-devel"
```
Comment (pullmoll) [inline srcpkgs/zimg/template]: This should most probably be `hostmakedepends="automake libtool pkg-config"`.
As mentioned elsewhere, `base-devel` does definitely not belong here.

**Comment by xtraeme** [comment]
- vapoursynth is missing a development pkg

**Pull request merged**